src/promptflow/promptflow/_sdk/_telemetry/activity.py

```diff
@@ -179,15 +184,19 @@ def log_activity(
             raise exception
 
 
-def extract_telemetry_info(self):
-    """Extract pf telemetry info from given telemetry mix-in instance."""
-    result = {}
+def extract_telemetry_info(telemetry_mixin):
+    """Extract pf telemetry info from given telemetry mix-in instance.
+    :param telemetry_mixin: telemetry mix-in instance.
+    :type telemetry_mixin: TelemetryMixin
+    :return: custom dimensions and user agent in telemetry.
+    :rtype: Tuple[Dict, Optional[str]]
+    """
     try:
-        if isinstance(self, TelemetryMixin):
-            return self._get_telemetry_values()
+        if isinstance(telemetry_mixin, TelemetryMixin):
+            return telemetry_mixin._get_telemetry_values(), telemetry_mixin._get_bonded_user_agent()
```

Comment:
Can we use another function to return **bonded_user_agent** without changing the return value format of this function?

Reply:
why this?

Reply:
Synchronized offline, let's keep it that way.